### ISSUE TITLE
Text Encoding

### ISSUE THREAD

**Issue by monolifed** [lesspass/lesspass#292]
I think it is UTF-8. But I couldn't find any information on the text encoding used for password (and other fields).

**Comment by guillaumevincent** [comment]
We use HMAC-SHA-256 with PBKDF2 to encrypt information from password profile.
HMAC operates on bytes, not text.
So texts are UTF-8 encoded string converted into a sequence of bytes.

I don't know why you ask this question.
If you want to implement LessPass core, there is one test with utf-8 encoded string char: https://github.com/lesspass/core/blob/master/test/pbkdf2.tests.js#L63-L76

**Comment by monolifed** [comment]
UTF-8 is the most obvious choice but I just wanted to be sure. Also, I thought it might be worth mentioning in the wiki.  
> If you want to implement LessPass core

I did some part of it as I mentioned in #175. Now, I got a somewhat working cli version. Though not very unicode friendly. (Currently it only gives consistent results on linux with terminal encoding set to UTF-8. I think it gives consistent results on windows if you use characters only from basic latin)
